sql-plugin/src/main/330+/scala/com/nvidia/spark/rapids/shims/GpuTypeShims.scala

```diff
@@ -160,4 +166,15 @@ object GpuTypeShims {
     }
   }
 
+  def isDayTimeInterval(dt: DataType) : Boolean = dt match {
```

Comment:
```suggestion
  def isDayTimeIntervalType(dt: DataType) : Boolean = dt match {
```
This can be simplified as 
```
def isDayTimeIntervalType(dt: DataType) : Boolean = dt.isInstanceOf[DayTimeIntervalType]
```